.openshift-ci/build/build-collector.sh

```diff
@@ -5,6 +5,13 @@ export DISABLE_PROFILING="true"
 export CMAKE_BUILD_DIR="$SRC_ROOT_DIR/cmake-build"
 export COLLECTOR_APPEND_CID=true
 
+# For all PRs build in debug mode, unless requested otherwise via labels.
+if is_in_PR_context; then
+    if ! pr_has_label "release-build-type"; then
```

Comment:
We can probable merge these 2 if conditions
```suggestion
if is_in_PR_context && ! pr_has_label "release-build-type"; then
```